grub2_theme_preview/cli.py

```diff
@@ -234,6 +234,10 @@ def _grub2_directory(platform):
     return  '/usr/lib/grub/%s' % platform
 
 
+def _grub2_platform():
+  return '%s-%s' % (platform.machine(), 'efi' if os.path.exists('/sys/firmware/efi') else 'pc')
```

Comment:
On semantic level:
```
In [1]: import platform 

In [2]: platform.machine()
Out[2]: 'x86_64'
```
```console
# uname -m
x86_64

# ls -1 /usr/lib/grub/
i386-pc
```
So on a `amd64` machine I'd need `i386` rather than `x86_64`. Am I missing something?

On code level:
- Please fix indentation to be four spaces (as the rest of the file) rather than two.
- (With picky hat on:) extracting two variables here may help readability, especially with that ternary around.

Reply:
Hello,
`amd64` or `x86_64` is `64-bit`, but `i386` is `32-bit`. I do not understand how your grub dir is `i386-pc`, while having a `64-bit` machine. I am guessing this is because of the executable types inside this directory. Can u conform on this?
```
# file /usr/lib/grub/x86_64-efi/acpi.mod
acpi.mod: ELF 64-bit LSB relocatable, x86-64, version 1 (SYSV), not stripped
```
My output for your commands is:
```
In [1]: import platform

In [2]: platform.machine()
Out[2]: 'x86_64'

In [3]: platform.processor()
Out[3]: 'x86_64'
```

```
# uname -m
x86_64

# ls -F /usr/lib/grub
grub-mkconfig_lib@  x86_64-efi/  x86_64-efi-signed/
```
Thanks for the code review :smile:. I'm making neccessary changes.